### ISSUE TITLE
version 6.0.4 not compatible with firebase 9

### ISSUE THREAD

**Issue by mralbobo** [FirebaseExtended/rxfire#85]
```
npm ERR! Could not resolve dependency:
npm ERR! peer firebase@"^10.0.0" from rxfire@6.0.4
npm ERR! node_modules/rxfire
npm ERR!   rxfire@"~6.0.3" from the root project
```

Would minor version bump be more appropriate than a patch? (6.1.0)

**Comment by davideast** [comment]
Apologies that was a mistake. I eagerly pushed the version up without setting a range. I'll push a `6.0.5` with a fix.

**Comment by davideast** [comment]
Fixed in #86!
